pages/marketplace/starters/index.jsx

```diff
@@ -61,50 +67,122 @@ export const getStaticProps = handleErrors(async ({ preview }) => {
+      <div className={s.firstPartyStarters}>
+        <div className={s.groupWrapper}>
+          <div className={s.intro}>
+            <h2>Full fledged demos</h2>
+            <p>
+              Use our pre-built demo projects to see all of DatoCMS&apos;s
+              features in a <strong>realistic production-ready setup</strong>.
+              Includes many example content types, and advanced features.
+            </p>
+          </div>
+          <section className={s.fullFledged}>
+            {fullFledged?.map((item) => (
+              <MarketplaceCard
+                key={item.code}
+                href={`/marketplace/starters/${item.code}`}
+                image={item.screenshot.responsiveImage}
+                technology={item.technology}
+                text={{
+                  title: item.name,
+                  description: item.cmsDescription,
+                }}
+                highlight={'Best choice to start!'}
+                badge={item.badge}
+                label={item.label}
+                size="large"
+              />
+            ))}
+          </section>
+        </div>
+
+        <div className={s.groupWrapper}>
+          <div className={s.intro}>
+            <h2>Tech starter kits</h2>
+            <p>
+              Kickstart your next project with our{' '}
+              <strong>official scaffolds</strong>. They offer all the best
+              practices to integrate DatoCMS with your frontend framework, with
+              minimal content and styling.
+            </p>
+          </div>
+          <section className={s.techStarters}>
+            {techStarters?.map((item) => (
+              <MarketplaceCard
+                key={item.code}
+                href={`/marketplace/starters/${item.code}`}
+                technology={item.technology.squareLogo || item.technology.logo}
+                text={{
+                  title: item.name,
+                  description: item.cmsDescription,
+                }}
+                badge={item.badge}
+                label={item.label}
+              />
+            ))}
+          </section>
+        </div>
+      </div>
+
+      <Wrapper>
+        <div className={s.intro}>
+          <h2>Community templates</h2>
+          <p>
+            Discover the projects created by the community. They are not
+            officially supported by DatoCMS, but they can be a great source of
+            inspiration to kickstart your next project.
+          </p>
+        </div>
+
         <Announce
           href="/docs/project-starters-and-templates#generate-a-project-starter-button"
           center
         >
           <strong>Want to create your own starter project?</strong> Learn how to
           do that in our documentation!
         </Announce>
-        <div className={s.grid}>
-          {starters?.map((item) => (
-            <PluginBox
-              title={item.name}
+
+        <div className={s.community}>
+          {community?.map((item) => (
```

Comment:
The `?.` shouldn't be necessary in fullFledged, starters and community :)

Reply:
Makes sense!